src/main/groovy/org/shipkit/internal/gradle/contributors/AllContributorsFetcherTask.java

```diff
@@ -32,7 +32,7 @@
     @OutputFile private File outputFile;
 
     /**
-     * See {@link ReleaseConfiguration.GitHub#getApiUrl()}
+     * See {@link org.shipkit.gradle.ReleaseConfiguration.GitHub#getApiUrl()}
```

Comment:
Thank you for fixing the link! I assume that this can also be fixed by adding a formal import.